feature_tests/src/structs.rs

```diff
@@ -375,8 +375,8 @@ pub mod ffi {
 
     impl StructArithmetic {
         #[diplomat::attr(auto, constructor)]
-        pub fn new(x: i32, y: i32) -> Self {
-            Self { x, y }
+        pub fn new(x: i32) -> Self {
```

Comment:
unclear why this is necessary, and it's ideal to limit the number of public api changes to feature_tests since it affects all the tested backends (and you may have to fix things in multiple spots)